examples/plots/Station_Plot.py

```diff
@@ -69,9 +69,7 @@
 
 # Map weather strings to WMO codes, which we can use to convert to symbols
 # Only use the first symbol if there are multiple
-wx_codes = {'': 0, 'HZ': 5, 'BR': 10, '-DZ': 51, 'DZ': 53, '+DZ': 55,
-            '-RA': 61, 'RA': 63, '+RA': 65, '-SN': 71, 'SN': 73, '+SN': 75}
-wx = [wx_codes[s.split()[0] if ' ' in s else s] for s in data['weather'].fillna('')]
+wx = [wx_code_map[s.split()[0] if ' ' in s else s] for s in data['weather'].fillna('')]
```

Comment:
Feels like this should be rolled in somewhere eventually?

Reply:
Sure, but there's no way I'm doing that right now. That's an exercise left once we have an actual data model.